mmstudio/src/main/java/org/micromanager/data/internal/multipagetiff/FileSet.java

```diff
@@ -182,8 +182,12 @@ public void writeImage(final Image imgIn) throws IOException {
          currentFrame_ = frame;
 
          omeMetadata_.addImageTagsToOME(img.getCoords(), img.getMetadata(),
-               ifdCount_, baseFilename_, currentTiffFilename_, currentTiffUUID_);
+                 ifdCount_, baseFilename_, currentTiffFilename_, currentTiffUUID_);
+      } catch (UnsupportedOperationException uoe) {
+         throw (uoe);
       } catch (Exception ex) {
+         // not sure why we continue when this fails, since there is also tiff related
+         // stuff that happens in the OME metadata code.
          ReportingUtils.logError(ex, "Problem writing OME metadata");
```

Comment:
My guess is just lazy coding (maybe it was too much work to propagate a checked exception?).